src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFailureDetails.tsx

```diff
@@ -1,31 +1,90 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import Button from 'app/components/button';
+import {Panel} from 'app/components/panels';
+import {IconCheckmark, IconWarning} from 'app/icons';
 import {t} from 'app/locale';
 
+import FooterWithButtons from './components/footerWithButtons';
+import HeaderWithHelp from './components/headerWithHelp';
+
+type ErrorDetail = {name: string; error: string};
+
 type Props = {
-  lambdaFunctionFailures: Array<{FunctionName: string; Runtime: string}>;
+  lambdaFunctionFailures: ErrorDetail[];
+  successCount: number;
 };
 
-export default function AwsLambdaFailureDetails({lambdaFunctionFailures}: Props) {
+export default function AwsLambdaFailureDetails({
+  lambdaFunctionFailures,
+  successCount,
+}: Props) {
+  return (
+    <React.Fragment>
+      <HeaderWithHelp docsUrl="https://docs.sentry.io/product/integrations/aws_lambda/" />
+      <Wrapper>
+        <ItemWrapper>
+          <StyledCheckmark isCircled />
+          <h3>{t('Succesfully updated %s functions', successCount)}</h3>
+        </ItemWrapper>
+        <ItemWrapper>
+          <StyledWarning />
+          <h3>{t('Failed to update %s functions', lambdaFunctionFailures.length)}</h3>
+        </ItemWrapper>
+        <StyledPanel>{lambdaFunctionFailures.map(SingleFailure)}</StyledPanel>
+      </Wrapper>
+      <FooterWithButtons buttonText={t('Finish Setup')} href="?finish_pipeline=1" />
+    </React.Fragment>
+  );
+}
+
+function SingleFailure(errorDetail: ErrorDetail) {
   return (
-    <Wrapper>
-      <h3>{t('Failed to update the following Lambda Functions')}</h3>
-      {lambdaFunctionFailures.map(func => {
-        return <div key={func.FunctionName}>{func.FunctionName}</div>;
-      })}
-      <StyledButton priority="primary" href="?finish_pipeline=1">
-        Finish
-      </StyledButton>
-    </Wrapper>
+    <StyledRow>
+      <span>{errorDetail.name}</span>
+      <Error>{errorDetail.error}</Error>
+    </StyledRow>
   );
 }
 
 const Wrapper = styled('div')`
-  margin: 20px;
+  margin: 100px 50px 50px 50px;
+`;
+
+const StyledRow = styled('div')`
+  padding: 16px;
+  background-color: ${p => p.theme.background};
+  display: flex;
+  flex-direction: column;
+  &:not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.innerBorder};
+    border-radius: inherit;
+  }
+`;
+
+const Error = styled('span')`
+  color: ${p => p.theme.gray300};
```

Comment:
Have you tested this page in dark mode?

Reply:
typically you want to avoid using direct colors

Reply:
@EvanPurkhiser What should I use instead? Actually, the `gray300` looks fine in dark mode:

<img width="608" alt="Screen Shot 2021-01-25 at 1 03 44 PM" src="https://user-images.githubusercontent.com/8533851/105765881-e704d800-5f0d-11eb-9b17-0e2c1a2f9a88.png">

Reply:
Should it be `subText`?

Reply:
Can we fix those borders too?

Reply:
Also just incase you didn't know, cmd+shif+l toggles dark light mode

Reply:
fixed